src/leader_scheduler.rs

```diff
@@ -19,10 +20,10 @@ use solana_vote_signer::rpc::LocalVoteSigner;
 use std::io::Cursor;
 use std::sync::Arc;
 
-pub const DEFAULT_BOOTSTRAP_HEIGHT: u64 = 1000;
-pub const DEFAULT_LEADER_ROTATION_INTERVAL: u64 = 100;
-pub const DEFAULT_SEED_ROTATION_INTERVAL: u64 = 1000;
-pub const DEFAULT_ACTIVE_WINDOW_LENGTH: u64 = 1000;
+pub const DEFAULT_BOOTSTRAP_HEIGHT: u64 = replay_stage::BLOCK_TICK_COUNT * 256;
+pub const DEFAULT_LEADER_ROTATION_INTERVAL: u64 = replay_stage::BLOCK_TICK_COUNT * 32;
+pub const DEFAULT_SEED_ROTATION_INTERVAL: u64 = replay_stage::BLOCK_TICK_COUNT * 256;
+pub const DEFAULT_ACTIVE_WINDOW_LENGTH: u64 = replay_stage::BLOCK_TICK_COUNT * 256;
```

Comment:
are these names in harmony with the book?  what's an active window?  ROTATION_INTERVAL should be SLOT_LENGTH, right?  should all of these also have TICK in their name, e.g. BOOTSTRAP_TICK_COUNT ?  Can we put BOOTSTRAP_TICK_COUNT in genesis somehow?